Aquality.Selenium.Template/Aquality.Selenium.Template/Forms/TopBarMenu.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Aquality.Selenium.Forms;
+using OpenQA.Selenium;
+using System.Collections.Generic;
+
+namespace Aquality.Selenium.Template.Forms
+{
+    public class TopBarMenu : Form
+    {
+        private readonly IDictionary<Item, string> menuItems = new Dictionary<Item, string>
+        {
+            { Item.ContactUs, "//div[@id='primary-navigation']//li[contains(@class, 'contact-us menu')]//a" }
+        };
+
+        public TopBarMenu() : base(By.Id("masthead"), "Header")
+        {
+        }
+
+        public void OpenHeaderMenu(Item menuItem)
```

Comment:
should we use short or full form for single-line methods?

Reply:
Yes, we can user short form